release/models/segment-routing/openconfig-rsvp-sr-ext.yang

```diff
@@ -379,9 +385,17 @@ module openconfig-rsvp-sr-ext {
     leaf last-calculated-time {
       type oc-types:timeticks64;
       description
-        "The time at which the last calculated value for bandwidth
-        utilisation was performed. The value is expressed relative
-        to the Unix Epoch (Jan 1, 1970 00:00:00 UTC).";
+        "The time at which the last calculated values for bandwidth
+        utilisation and segment routing traffic was performed. 
+        The value is expressed relative to the Unix Epoch 
+        (Jan 1, 1970 00:00:00 UTC).";
+    }
+    
+    leaf last-calculated-sr-traffic {
+      type uint64;
+      units "kbps";
+      description
+        "The last calculated value of the Segment Routing traffic.";   
```

Comment:
Is this the same as "last-calculated-sr-utilisation" except that it is specifying the units?    

Should last-calculated-sr-utilisation instead be updated to specify units "kbps".  (Apologies if I am missing reference to segment routing)

Reply:
Yes, it looks like it -- I think the original issue raised here was an artifact of the fact that the name of this leaf got updated in review, and we didn't catch it.

One observation - although it pains me - is that this is the British English spelling of utilisation, I think the standard across the models today is American English, so we maybe should update the leaf name.